tests/kola/console/console-login-helper-service

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# This is RHCOS only
+
+# kola: { "exclusive": false }
+set -xeuo pipefail
+
+ok() {
+    echo "ok" "$@"
+}
+
+fatal() {
+    echo "$@" >&2
+    exit 1
+}
+
+# Check that we have the proper presets for console-login-helper-messages in RHCOS.
+# Versions of CLHM prior to v0.21 have `issuegen`-related systemd units.
+if test -f /usr/lib/systemd/system/console-login-helper-messages-issuegen.service; then
```

Comment:
This probably should always exist. Noting here for reference. Can be in a follow up PR.